src/main/java/seedu/address/logic/commands/EditRentalCommand.java

```diff
@@ -69,7 +70,8 @@ public class EditRentalCommand extends Command {
     public static final String MESSAGE_NOT_EDITED = "At least one field to edit must be provided.";
     public static final String MESSAGE_DUPLICATE_RENTAL = "This client already exists in the address book.";
     public static final String MESSAGE_DUPLICATE_RENTAL_INFORMATION =
-            "The editing rental information already exists in the address book";
+            "The editing rental information already exists in the address book.";
+    public static final String MESSAGE_INVALID_RENTAL_DATES = "Rental end date should be after rental start date.";
```

Comment:
Perhaps DATE is sufficient. Nothing serious

Reply:
There is also rent due date, so the name is to differentiate them.